app/models/suggestion.rb

```diff
@@ -0,0 +1,14 @@
+class Suggestion < ApplicationRecord
+  validates :title, :description, presence: true
```

Comment:
Add a presence validation for `:user` also, so a Suggestion could not save unless it is associated with a User